src/hyperion/external_interaction/callbacks/xray_centre/ispyb_callback.py

```diff
@@ -115,7 +120,7 @@ def activity_gated_event(self, doc: Event):
             )
 
         set_dcgid_tag(self.ispyb_ids.data_collection_group_id)
-        return doc
+        return self._tag_doc(doc)
```

Comment:
Should: We've already called `super().activity_gated_event(doc)` here, so we already have the tag in there?